synapse_invite_checker/invite_checker.py

```diff
@@ -319,8 +327,9 @@ async def fetch_federation_allow_list(self) -> set[str]:
             msg = "Empty federation list"
             raise Exception(msg)
 
-        fedlist = FederationList.model_validate_json(jws_verify.payload)
-        return {fed.domain for fed in fedlist.domainList}
+        # Make sure to validate incoming, potentially incomplete or corrupt data
```

Comment:
```suggestion
        # Validate incoming, potentially incomplete or corrupt data
```

Reply:
I think the other words are just filler here

Reply:
Yes this is not a school essay. Unnecessary words are unnecessary. They can go

Reply:
fixed in e64ae8f